app/validator.py

```diff
@@ -0,0 +1,133 @@
+import argparse
+import logging
+import redis
+import time
+import sys
+from queuelogger import QueueLogger
+from datetime import datetime
+from rq import Queue, Connection, Worker
+from jobstatus import JobState
+from config import Config
+
+# Logger
+LOGGER = logging.getLogger(__name__)
+QUEUELOGGER = QueueLogger(1)
+sys.stdout = QUEUELOGGER
+sys.stderr = QUEUELOGGER
+
+class Validator(object):
+    """
+    Validates the state of all job status records and re-queues any jobs that ran into processing problems.
+    """
+
+    def __init__(self, logger, redisHost, redisPort):
+        """
+        Class constructor.
+
+        :param logger logger: the logger
+        :param str redis_host: Redis host where the Redis Q is running
+        :param int redis_port: Redis port where the Redis Q is running
+        """
+        self.logger = logger
+        self.config = Config()
+        self.redis_host = redisHost
+        self.redis_port = redisPort
+    
+    def requeue_job(self, job_id):
+        """
+        Requeues a job for processing
+
+        :param str job_id: Id of the job that needs to be re-processed
+        """
+        # TODO: Requeue job
+
+    def validate_job_health(self, job_key, redis_conn):
+        """
+        Validates the health of a job based on the job state and life timespan
+
+        :param str job_key: Key to retrieve job status
+        :param object redis_conn: Redis connection object
+        """
+        # get the job from redis
+        jobStatus = redis_conn.get(job_key)
+
+        if(jobStatus is not None):
```

Comment:
Also use `!=` here
If you know `jobStatus` is never `0` or False then you can use `if(jobStatus)`